libpldmresponder/platform.hpp

```diff
@@ -465,6 +493,9 @@ class Handler : public CmdHandler
     pldm::responder::oem_platform::Handler* oemPlatformHandler;
     sdeventplus::Event& event;
     std::string pdrJsonsDir;
+    std::atomic<pldmRepoStatus> repoCreated;
+
+    std::unique_ptr<sdeventplus::source::Defer> deferredPDRGeneration;
     bool pdrCreated;
```

Comment:
This must be std::atomic<bool> pdrCreated as it's used concurrently across threads.